metrics/src/counter.rs

```diff
@@ -313,31 +240,31 @@ mod tests {
 
     #[test]
     #[serial]
-    fn test_high_rate_counter() {
+    fn test_metrisrate() {
```

Comment:
"metris" rate?

Reply:
Yeah, it's that new Tetris spinoff everyone's been raving about!